### ISSUE TITLE
My NVDA is conflicting with my D-Link connection manager

### ISSUE THREAD

**Issue by Yanyan28** [nvaccess/nvda#7665]
Whenever I turn on my D-Link connection manager my NVDA stops reading contents inside the My Documents folder on my PC. Can someone help me please.

**Comment by derekriemer** [comment]
can you explain how you reproduce this issue?

**Comment by Yanyan28** [comment]
I am using a D-Link DWM-221 4G LTE USB Adapter.  A portable travel modem that allows me to stay connected to the Internet wherever you I am. The dongle is installed with a connection manager. Problem is when I need to use the dongle and open the connection manager, my NVDA cannot read the contents inside My Documents folder on my PC. I have already tried to re-install the connection manager. But nothing happened.

**Comment by derekriemer** [comment]
to be clear, does the documents folder becomes inaccessible, after the software is installed, or after the software is running?

**Comment by Yanyan28** [comment]
Folder becomes inaccessible after the software is running.

**Comment by derekriemer** [comment]
Someone should look into this. I'm still not sure if we can download the software free of charge from somewhere.

**Comment by Yanyan28** [comment]
Are there no available steps that I can try to fix the issue?Like running codes on CMD or something?

**Comment by derekriemer** [comment]
we'd need to get ahold of the D-Link connection manager software. Can you try giving us some idea of what won't read in your documents folder, I.E. what elements can you see with NVDA? Is it visually workable, Can you reproduce it with any other screen readers, and what windows version?

**Comment by Adriani90** [comment]
@Yanyan28 are you still having this problem in NVDA 2019.1.1?

**Comment by Adriani90** [comment]
We don't have updates on this issue since many years. I am closing as abandoned. Please comment on this if you are still having this issue and we can reopen it. Thanks.
